src/components/ResultList/Item/Item.tsx

```diff
@@ -84,30 +84,29 @@ export const Item = (props: IItemProps): ReactElement => {
         )}
       </div>
       <div className="flex flex-col mb-1 mr-3 w-full">
-        <div className="flex flex-row justify-between pr-2 py-2">
+        <div className="flex flex-row justify-between pr-2 pt-1">
           <Link href={`/abs/${bibcode}`}>
-            <a className="text-blue-700 hover:underline">
-              <h3 className="text-lg" id={`result-${id}`} dangerouslySetInnerHTML={{ __html: title[0] }}></h3>
+            <a className="pt-1 text-blue-700 hover:underline">
+              <h3 className="text-lg leading-6" id={`result-${id}`} dangerouslySetInnerHTML={{ __html: title[0] }}></h3>
             </a>
           </Link>
-          <div className="flex items-start">{hideActions ? null : <ItemResourceDropdowns doc={doc} />}</div>
+          <div className="flex items-start ml-2">{hideActions ? null : <ItemResourceDropdowns doc={doc} />}</div>
         </div>
         <div className="flex flex-col">
           {author.length > 0 && (
-            <div className="text-s">
+            <div className="text-sm">
               {author.slice(0, 10).join('; ')}
               {author_count > 10 && <span className="text-xs italic"> and {author_count - 10} more</span>}
             </div>
           )}
           <div className="flex py-1">
             <span className="text-xs">
               {formattedPubDate}
-              {formattedPubDate && formattedBibstem ? ' | ' : ''}
+              {formattedPubDate && formattedBibstem ? <>&nbsp;&nbsp;·&nbsp;&nbsp;</> : ''}
```

Comment:
maybe dumb question by is this better than doing:
```suggestion
              {formattedPubDate && formattedBibstem ? <span className="px-1">·</span> : ''}
```
or whatever padding would look good

Reply:
You are right!

Reply:
Changed